### DISCUSSION TITLE
is dbus-serialbattery limiting discharge current?

### DISCUSSION THREAD

**Discussion by dominikfe** [Louisvdw/dbus-serialbattery#203] [Q&A] [answered]
Hi!
I am seeing something limiting the discharge current at 30% and then again at 20% SoC.
I cant imagine what part of my installation is causing this limitation as I did not configure anything like this somewhere.
Thank you!

**Comment by Louisvdw** [comment, marked as answer]
Look at the CCCM feature 
https://github.com/Louisvdw/dbus-serialbattery/wiki/Features#charge-current-control-management

You can disable it on line 25 of utils.py with
CCCM_ENABLE = False

**Reply by dominikfe** [reply]
Sorry, was about to delete the post as I found it. THX for the rapid reply!
